### PR TITLE
Format stat message

### PR DESCRIPTION
**Pull request by AZatsepa**
Screenshot from 2019-07-04 23-40-34

### CODE REVIEW

**Review by yashka713 — APPROVED** [review]
:+1:

**Comment by yashka713** [comment]
This PR need to be changed.

File: app/services/karma_service.rb [hunk 1]
```diff
@@ -14,22 +14,52 @@ def show_statistic
...
+    until name.length == 20 do name += ' ' end
+    name
+  end
+
+  def user_statistic
     medals = %w[🥇 🥈 🥉]
-    result = '🏆This is the karma champions list🏆'
-    DB.users.each { |user| result << user_statistic(user, medals) }
-    result
+    ''.yield_self do |result|
```
Comment (TheBlackArroVV) [inline app/services/karma_service.rb:40]: You can use `then` instead of `yield_self`. I think it's more readeable

Comment (AZatsepa) [reply app/services/karma_service.rb:40]: Ruby 2.5.0 doesn't have a method `then`

**Review by TheBlackArroVV — APPROVED** [review]

**Pull request merged**